docs/osquery/exported-fields-reference.asciidoc

```diff
@@ -2236,7 +2250,7 @@ For more information about osquery tables, see the https://osquery.io/schema[osq
 
 *hostname* - keyword, text.text
 
-* _curl_certificate.hostname_ - Hostname (domain[:port]) to CURL
+* _curl_certificate.hostname_ - Hostname to CURL (domain[:port], e.g. osquery.io)
```

Comment:
```suggestion
* _curl_certificate.hostname_ - Hostname to CURL (domain[:port], for example, osquery.io)
```

Reply:
We usually try to avoid using abbreviations since these are not always understsood by nonnative-English-speaking folks.